recipes/qt/5.x.x/conanfile.py

```diff
@@ -930,9 +932,9 @@ def _create_private_module(module, dependencies=[]):
     def package_id(self):
         del self.info.options.cross_compile
         del self.info.options.sysroot
-        if self.options.multiconfiguration and is_msvc(self):
-            if self.settings.compiler == "Visual Studio":
-                if "MD" in self.settings.compiler.runtime:
+        if self.info.options.multiconfiguration and is_msvc(self):
```

Comment:
it's not v2 compatible when multiconfiguration=True because is_msvc() is not allowed in package_id()